content/client-lib-development-guide/features.textile

```diff
@@ -974,9 +984,11 @@ h4. Stats
 
 h4. ErrorInfo
 
-* @(TI1)@ Provides a generic Ably @ErrorInfo@ object that contains Ably @code@, @statusCode@ (analogous to HTTP status code) and @message@ attributes
+* @(TI1)@ Provides a generic Ably @ErrorInfo@ object that contains Ably @code@, @statusCode@ (analogous to HTTP status code), @message@ and @source@ attributes
```

Comment:
I don't think you were part of the conversation we had a while, and I am unable to find the issue unfortunately, but we have plenty of situations where the error sent to the user does not accurately reflect the underlying reason. So we suggested having a source attribute in errors sent from Ably that can be used to indicate the underlying source reason for the error.  So only Ably, at present, will use this.